### ISSUE TITLE
Feature Request: Ability to specify multiple formatters

### ISSUE THREAD

**Issue by shepmaster** [brigade/scss-lint#571]
In CI, I am using the Checkstyle formatter, but (for reasons unrelated to this project), Jenkins does not show the actual errors. When using Rubocop, I worked around this by outputting the standard output to the console *and* outputting the checkstyle information to a file.

As a workaround, we are simply running `scss-lint` twice, once with a formatter and once without. This does entail extra overhead.

**Comment by sds** [comment]
Hey @shepmaster, thanks for suggesting ways we can improve the tool.

Fortunately, this feature is already supported. See #273.

**Comment by shepmaster** [comment]
@sds Oops, sorry. Could you help me figure out where I should have learned that?

I did try `scss-lint --require 'scss_lint_reporter_checkstyle' --format Default --format 'Checkstyle' --out 'foo.xml'`, but that does **not** output the default formatter. Neither does `scss-lint --require 'scss_lint_reporter_checkstyle' --format Default --out :stdout --format 'Checkstyle' --out 'foo.xml'`. In both cases, the checkstyle output is present in the file.

I'm happy to submit a PR to the README to mention how to do this, once I figure out how to make it work. :smiley_cat:

**Comment by sds** [comment]
Hmm, I agree this is confusing (we don't make use of this feature at all internally, so that's probably why it's a bit janky).

FWIW, I was able to get this to work with:

```bash
scss-lint --require scss_lint_reporter_checkstyle --format Checkstyle --out foo.xml --format Default
```

I'm more than happy to accept a pull request that better documents this capability!
